Userland/Libraries/LibJS/Runtime/Date.h

```diff
@@ -68,6 +71,9 @@ class Date final : public Object {
     String time_string() const { return m_datetime.to_string("%T GMT+0000 (UTC)"); }
     String string() const
     {
+        if (is_invalid())
+            return String { "Invalid Date" };
+
```

Comment:
You can use the implicit variant here:

```suggestion
        if (is_invalid())
            return "Invalid Date";
```